docs/client-development/mandate.rst

```diff
@@ -0,0 +1,520 @@
+                "services": [{
+                    "id": "urn:etoegang:DV:00000001002308836000:services:9113",
+                    "uuid": "34085d78-21aa-4481-a219-b28d7f3282fc"
+                }]
+            }
+        }
+    }
+
+
+**eHerkenning - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Persoon waarover bewind gevoerd wordt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "inpBsn": "111222333"
+        },
+        "authenticatieContext": null
+    }
+
+
+5. eHerkenning / With mandate (ketenmachtiging)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**eHerkenning - initiator**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+        "source": "eherkenning",
+        "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+        "representee": {
+            "identifierType": "kvkNummer",
+            "identifier": "99998888"
+        },
+        "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+        "mandate": {
+            "services": [{
+                "id": "urn:etoegang:DV:00000001002308836000:services:9113",
+                "uuid": "34085d78-21aa-4481-a219-b28d7f3282fc"
+            }]
+            }
+        }
+    }
+
+
+**eHerkenning - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "niet_natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Bedrijf dat de intermediair machtigt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "99998888"
+        },
+        "authenticatieContext": null
+    }
+
+
+Query patterns
+--------------
+
+Below are examples how to request zaken, authorized by different parties.
+
+**DigiD**
+
+1. Show me my cases (based on my BSN) opened for myself:
+
+    .. code::
+
+        POST /zaken/api/v1/zaken?
+             rol__betrokkeneIdentificatie__natuurlijkPersoon__inpBsn=<ownBsn>
```

Comment:
Minor nitpick, but i don't like `ownBsn` being in camel case. I think `<own bsn>` or a different name would be better.

Reply:
Also is it a POST request that is using the GET parameters format?

Reply:
Another Comment: should these also use the http code block like the above ones

Reply:
Ooooh, all of requests in this section should be GET, I don't know why I used POST here, I'll fix it

I omitted `http` code block here, because Sphinx http validator kept complaining about the format, because I separated a request into several lines. 
I haven't found how you can do it with HTTP syntax which Sphinx would be ok with and I didn't want to put all query parameters into one line, because it won't be readable, so I gave in and decided to remove "http" from "code" derictive here. 

All of the query parameters and in camel case, so I thought camel case would be ok here, I can replace it with snake case, and make it `<own_bsn>.  I really don't want to use space though. Should I make it a snake case?

Reply:
I do think snake case is better to differentiate it from the parameters.

![Screenshot from 2024-10-30 11-00-42](https://github.com/user-attachments/assets/fe05e495-305a-4f34-a44d-e2d89c518f2b)

Reply:
changed all query parameters (ownBsn, ownRsin and ownKvk) to snake case